src/components/markdown/textarea.vue

```diff
@@ -14,14 +14,14 @@ except according to the terms contained in the LICENSE file.
     <div class="form-group">
       <textarea :value="value" class="form-control"
         :placeholder="defaultText" :aria-label="defaultText"
-        :required="required" rows="2" @input="$emit('input', $event.target.value)">
+        :required="required" :rows="rows" @input="$emit('input', $event.target.value)">
       </textarea>
     </div>
     <div v-if="value !== '' && previewMode" class="preview-container">
```

Comment:
Is this another place where `emptyValue` is needed? Or is it impossible for `value` to be `null` if `previewMode` is `true`? (In that case, maybe a comment would be useful?)

Reply:
emptyValue is needed here, yes!